Consume new code

TimMikeladze opened the pull request.

pradel commented on README.md:
```diff
@@ -1,24 +1,7 @@
 
-This example demonstrate how to use [js-accounts](https://github.com/js-accounts/accounts) with [express](http://expressjs.com/), [mongodb](https://mongodb.com/), [react](https://facebook.github.io/react/) and [material-ui](http://www.material-ui.com/).
+The examples demonstrate how to setup the client and server side packages with different transports.
 
```
Maybe add a link list  to point to each packages

pradel commented:
The rest-example is now working.
There is still some bugs:
* When you hit a bad password json.parse is failing.
* When you login you will see the user info. If you refresh the page it looks like the tokens are deleted and you are not logged in anymore.

pradel commented:
@TimMikeladze example is ready!

TimMikeladze commented:
Thanks @pradel. The example looks good. Running into an issue in our work app: I updated the packages to the latest versions and modified the code to manually resume the session like in the example but am getting a `Uncaught (in promise) Error: no tokens provided` error. This could easily (and most likely) be something with the app and not the accounts package but I haven't investigated yet. Just a heads up.

pradel commented:
@TimMikeladze Got the same in my app but didn't have the time to find why yet. I will try to find in the next days.

pradel commented:
Here is the example for the redesign of the client.
It use react-router v4.
